sprout/appliances/views.py

```diff
@@ -202,13 +202,18 @@ def versions_for_group(request):
         except ObjectDoesNotExist:
             versions = []
         else:
-            versions = Template.get_versions(
-                container_q,
-                template_group=group, ready=True, usable=True, exists=True,
-                preconfigured=preconfigured, provider__working=True, provider__disabled=False,
-                provider__user_groups__in=request.user.groups.all())
+            versions = [
+                (version, Template.ga_version(version))
```

Comment:
why not only call ga_version on the actual version you take? also a limit might make sense